robot/EDA/resources/EDA.py

```diff
@@ -11,10 +11,12 @@
 from selenium.common.exceptions import WebDriverException
 from selenium.webdriver.common.keys import Keys
 
+from locators_53 import eda_lex_locators as locators_53
 from locators_51 import eda_lex_locators as locators_51
 from locators_50 import eda_lex_locators as locators_50
```

Comment:
Not necessarily in this PR, but you can eventually remove the `locators_50` file.